mappings/net/minecraft/server/network/ServerPlayerEntity.mapping

```diff
@@ -13,7 +13,7 @@ CLASS vf net/minecraft/server/network/ServerPlayerEntity
 	FIELD cp seenCredits Z
 	FIELD cq recipeBook Lym;
 	FIELD cu enteredNetherPos Lcrg;
-	FIELD cv chunkPos Lfo;
+	FIELD cv lastCameraChunkSectionPosition Lfo;
```

Comment:
In what sense is this the last position?

Reply:
It's the position of the last camera update. It's checked and set by `ThreadedAnvilChunkStorage.updateCameraPosition` to check if the player changed chunks aince last tick.

Reply:
Ah ok, that makes sense. 👍